src/index.html

```diff
@@ -0,0 +1,42 @@
+<h1>
+  Calcite App Components
+</h1>
+<p>A collection of calcite components for building single page applications.</p>
```

Comment:
Can we add a link to the demo folder or a small paragraph on how to get to the demo pages?

Reply:
Yep. added!